kernel_gateway/jupyter_websocket/__init__.py

```diff
@@ -50,7 +50,7 @@ def create_request_handlers(self):
             handlers.append(new_handler)
         return handlers
 
-    def should_seed_cell(self, code):
+    def should_seed_cell(self, cell):
```

Comment:
Is it a code string or a cell object? The docstring should have a `Parameters` section describing the param and its type plus the return value of the method. (Same goes for the template above if you adopt it and the other code -> cell change below.)